lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<ul><li>"které jsou nad rámec tohoto kurzu" bych vyměnil za "které nepotřebujeme, protože chceme data z API pouze získávat"</li><li>možná bych doplnil, že adresa samotná nejčastěji určuje o jaká data půjde (v příkladu jsou to filmy) a URL parametry umožňují provést filtraci už na straně serveru a získat tím jen ta data, která nás opravdu zajímají.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>doplněno</p>